example/components/StyledFirebaseAuth.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Auth, onAuthStateChanged } from 'firebase/auth'
+import 'firebaseui/dist/firebaseui.css'
+import { auth } from 'firebaseui'
+
+interface Props {
+  // The Firebase UI Web UI Config object.
+  // See: https://github.com/firebase/firebaseui-web#configuration
+  uiConfig: auth.Config
+  // Callback that will be passed the FirebaseUi instance before it is
+  // started. This allows access to certain configuration options such as
+  // disableAutoSignIn().
+  uiCallback?(ui: auth.AuthUI): void
+  // The Firebase App auth instance to use.
+  firebaseAuth: Auth // As firebaseui-web
+  className?: string
+}
```

Comment:
This lint rule is no longer relevant. Could you add `"react/require-default-props": "off"` to the .eslintrc.js rules?

Reply:
Done!

> Reminder that you can check linting yourself locally by running yarn lint at the package root.

Thanks, I wasn't aware. I'm usually not working with `eslint`, so this was mostly trial & error.